test/katex-spec.js

```diff
@@ -2999,11 +2999,11 @@ describe("The \\mathchoice function", function() {
 
 describe("Symbols", function() {
     it("should parse \\text{\\i\\j}", () => {
-        expect("\\text{\\i\\j}").toParse();
+        expect("\\text{\\i\\j}").toBuild();
```

Comment:
Just curious, why did these change?

Reply:
No reason; it would be fine as is, too.  But `toBuild` also checks `toParse`, so this seemed like a better test.